### DISCUSSION TITLE
Is there a way to export the whole document website as a pdf?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#7452] [Q&A]
As the title mentioned, I want to know which plugin is most suitable with mkdocs-material?

Thanks!

**Comment by alexvoss** [comment]
From the [example](https://github.com/orzih/mkdocs-with-pdf/blob/master/samples/mkdocs-material/document.pdf) posted, it seems that [mkdocs-with-pd](https://github.com/orzih/mkdocs-with-pdf) produces quite usable results. It seems abandoned, though. 

Another approach would be to see how far you get with [Pandoc](https://pandoc.org/) compiling your Markdown to PDF. Again, your mileage may vary depending on how well your Markdown code matches [a dialect that Pandoc can process](https://pandoc.org/chunkedhtml-demo/8.22-markdown-variants.html).

**Reply by kamilkrzyskow** [reply]
There is also https://timvink.github.io/mkdocs-print-site-plugin/
